examples/complex.c

```diff
@@ -166,9 +168,9 @@ static int sockopt_cb(xmpp_conn_t *conn, void *socket)
             return ret;
 #endif /* TCP_USER_TIMEOUT */
     }
-
 #endif /* _WIN32 */
-    return ret;
+
+    return 0;
```

Comment:
was this change intentional?

Reply:
All error handling is done immediately after it could occur or not? i.e. we would only arrive here when `ret` is `0`!?

Reply:
I reverted it in the final squash because that is what is returning success/fail for the _WIN32 case (same is in sock.c/sock_set_keepalive())

Reply:
then it was accidental :smile: